### PR TITLE
enable android

### PR DESCRIPTION
**Pull request by aep**

### CODE REVIEW

File: src/platform/mod.rs [hunk 1]
```diff
@@ -33,6 +33,11 @@ pub mod linux;
 #[cfg(target_os = "linux")]
 pub use self::linux::PlatformImpl;
 
+#[cfg(target_os = "android")]
+pub mod linux;
+#[cfg(target_os = "android")]
+pub use self::linux::PlatformImpl;
```
Comment (myfreeweb) [inline src/platform/mod.rs]: Please use `any()` here too

File: Cargo.toml [hunk 2]
```diff
@@ -18,6 +18,8 @@ libc = "0.2"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 nom = "3.2"
+[target.'cfg(target_os = "android")'.dependencies]
```
Comment (myfreeweb) [inline Cargo.toml]: I think `any()` should work here too

Comment (aep) [reply Cargo.toml]: nice. didnt know. fixed

**Comment by myfreeweb** [comment]
Thanks!

**Pull request merged**